src/cardboard-distorter.js

```diff
@@ -98,8 +98,7 @@ function createRenderTarget(renderer) {
   var parameters = {minFilter: THREE.LinearFilter,
                     magFilter: THREE.LinearFilter,
                     format: THREE.RGBFormat,
-                    stencilBuffer: false,
-                    depthBuffer: false};
```

Comment:
There isn't a need for depthBuffer for the distortion, however it is needed for that [textureTarget](https://github.com/borismus/webvr-boilerplate/blob/32cd6f5/src/cardboard-distorter.js#L135).

Unless I missed something, the `createRenterTarget` function is currently used for the `genuineRender` target (the original render function renders to it when in VR mode), so it does create problems if it doesn't have a depth buffer. The ShaderPass then renders using the same `genuineRender` function but using the default target ([cardboard-distorter.js#L168](https://github.com/borismus/webvr-boilerplate/blob/32cd6f5/src/cardboard-distorter.js#L168) and [#L91](https://github.com/borismus/webvr-boilerplate/blob/32cd6f5/src/cardboard-distorter.js#L91)).

I just encountered that problem, where the non-cardboard view renders fine, but as soon as you switch to cardboard view some objects that should be behind others appear in front (issue #55 is very related).
Removing the `depthBuffer: false` solves it.

So https://github.com/borismus/webvr-boilerplate/commit/32cd6f543534960a104e2df9d247c451b74f9e01 actually re-introduce the issue, and #55 isn't fixed as a result.

The distorter doesn't need a depthBuffer, but that `textureTarget` is used as output for genuine rendering and input for the distortion. It does need a depthBuffer, otherwise the genuine rendering doesn't render properly.
Can we revert https://github.com/borismus/webvr-boilerplate/commit/32cd6f543534960a104e2df9d247c451b74f9e01 please?